docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,63 @@
+# Release 363 (XX Oct 2021)
+
+## General
+
+* Reduce query memory usage when spill occurs. ({issue}`9270`)
+* Reduce join memory usage. ({issue}`9327`)
+* Reduce query memory usage during join unspilling. ({issue}`9275`)
```

Comment:
#9270 is about unspilling too.  merge?

Reply:
yup. should be both https://github.com/trinodb/trino/pull/9275